extensions/extension_types.yaml

```diff
@@ -27,7 +32,11 @@ types:
   - name: multipolygon
     structure:
       T: List<polygon>
+  - name: mixed
+    parameters:
+      - name: T
+        type: union<crs_point, linestring, polygon, multipoint, multilinestring, multipolygon>
   - name: geometry
     parameters:
       - name: T
-        type: union<point, linestring, polygon, multipoint, multilinestring, multipolygon>
+        type: List<mixed>
```

Comment:
Geometry should allow for individual items as well based on the usage we're seeing.   Perhaps we need both mixed and List<mixed> here?

Reply:
hmm....so 2 geometry types? one list<mixed> and one just mixed?  Or is there a way for it being one or the other?

Reply:
A union would allow it to contain both lists and non-lists.  I'm now second guessing if we want an intermediate type though.  Might just be cleaner for one level of union.